src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -286,14 +288,142 @@ def create_certificate(client, vault_base_url, certificate_name, certificate_pol
+    :param issuer_name: Unique identifier for the issuer settings.
+    :param provider_name: The certificate provider name. Must be registered with your
+        tenant ID and in your region.
+    :param account_id: The issuer account id/username/etc.
+    :param password: The issuer account password/secret/etc.
+    :param organization_id: The organization id.
+    """
+    from azure.cli.command_modules.keyvault.keyvaultclient.generated.models import \
+        (CertificateIssuerSetParameters, IssuerCredentials, OrganizationDetails, IssuerAttributes,
+         AdministratorDetails, KeyVaultErrorException)
+
+    def update(obj, prop, value, nullable=False):
+        set_value = value if value is not None else getattr(obj, prop, None)
+        if not set_value and not nullable:
+            raise CLIError("property '{}' cannot be cleared".format(prop))
+        elif not set_value and nullable:
+            set_value = None
+        setattr(obj, prop, set_value)
+
+    issuer = client.get_certificate_issuer(vault_base_url, issuer_name)
+    update(issuer.credentials, 'account_id', account_id, True)
+    update(issuer.credentials, 'password', password, True)
+    update(issuer.attributes, 'enabled', enabled)
+    update(issuer.organization_details, 'id', organization_id, True)
+    update(issuer, 'provider', provider_name)
+    return client.set_certificate_issuer(
+        vault_base_url, issuer_name, issuer.provider, issuer.credentials,
+        issuer.organization_details, issuer.attributes)
+
+def list_certificate_issuer_admins(client, vault_base_url, issuer_name):
+    """ List admins for a specified certificate issuer. """
+    return client.get_certificate_issuer(
+        vault_base_url, issuer_name).organization_details.admin_details
+
+def add_certificate_issuer_admin(client, vault_base_url, issuer_name, email, name=None,
+                                 phone=None):
+    """ Add admin details for a specified certificate issuer. """
+    from azure.cli.command_modules.keyvault.keyvaultclient.generated.models import \
+        (AdministratorDetails, KeyVaultErrorException)
+    issuer = client.get_certificate_issuer(vault_base_url, issuer_name)
+    org_details = issuer.organization_details
+    admins = org_details.admin_details
+    first_name = None
+    last_name = None
+    if name:
+        name_comps = name.rsplit(' ', 1)
+        first_name = name_comps[0] if len(name_comps) > 1 else name_comps
+        last_name = name_comps[-1] if len(name_comps) > 1 else None
+    if any((x for x in admins if x.email_address == email)):
+        raise CLIError("admin '{}' already exists".format(email))
+    new_admin = AdministratorDetails(first_name, last_name, email, phone)
+    admins.append(new_admin)
+    org_details.admin_details = admins
+    client.set_certificate_issuer(
+        vault_base_url, issuer_name, issuer.provider, issuer.credentials, org_details,
+        issuer.attributes)
+    return {'newAdmin': new_admin}
```

Comment:
Looks like there's a bug in `_encode_hex` then as in python 2, a str is an instance of bytes (as they are just synonyms in python 2) so it gets converted with `return base64.b64encode(item).decode('utf-8')`:
e.g.
```
>>> a = 'helloworld'
>>> type(a)
<type 'str'>
>>> isinstance(a, bytes)
True
```

"The new bytes type is 3.x only. The 2.x bytes built-in is just an alias to the str type."
from: http://stackoverflow.com/a/5901825

I'd rather see `_encode_hex` fixed if that's the only reason for returning `{'newAdmin': new_admin}` here.

Under what conditions do you want to run the b64encode?